lesson02/main.js

```diff
@@ -1 +1,26 @@
 "use strict";
+
+const ul = document.getElementById("js-target");
+
+const li = document.createElement("li");
+
+const a = document.createElement("a");
+a.href = "1.html";
+a.textContent = "これです";
+
+const img = document.createElement("img");
+img.src = "bookmark.png";
+img.alt = "ブックマーク";
+
```

Comment:
```suggestion
const a = document.createElement("a");
const img = document.createElement("img");
a.href = "1.html";
a.textContent = "これです";
img.src = "bookmark.png";
img.alt = "ブックマーク";

```
It's a minor detail, but it's easier to read if it's separated this way.

Reply:
Thank you sen-san!
l'll write the definition at first .